hooks/lib/pdlc-test-strategy.sh

```
@@ -0,0 +1,145 @@
+  if [[ -f "$plan_file" ]]; then
+    has_plan=1
+    # Try to detect test framework from plan
+    local detected_framework
+    detected_framework=$(grep -ioE '(BATS|Jest|Mocha|pytest|JUnit|RSpec|Go test)' "$plan_file" 2>/dev/null | head -1) || true
+    if [[ -n "$detected_framework" ]]; then
+      test_framework="$detected_framework"
+    fi
+  fi
+
+  # ── Determine recommended test types ────────────────────
+  local recommend_unit=1
```

Shellcheck (0.11.0)

[warning] recommend_unit appears unused. Verify use (or export if used externally).

(SC2034)